packages/server/tests/localAcceptance.env

```diff
@@ -27,7 +27,7 @@ LIMIT_DURATION=90000
 SERVER_REQUEST_TIMEOUT_MS=60000
 MEMWATCH_ENABLED=true
 WRITE_SNAPSHOT_ON_MEMORY_LEAK=false
-HBAR_RATE_LIMIT_TINYBAR=5000000000# 50 HBARs
-HBAR_RATE_LIMIT_DURATION=80000# 80 seconds
-HBAR_RATE_LIMIT_BASIC=4000000000# 40 HBARs
+HBAR_RATE_LIMIT_TINYBAR=9000000000# 90 HBARs
+HBAR_RATE_LIMIT_DURATION=180000# 180 seconds
+HBAR_RATE_LIMIT_BASIC=6000000000# 60 HBARs
```

Comment:
**Note for Reviewers:** 
- The values of HBAR_RATE_LIMIT_TINYBAR has been increased due to the addition of more exhaustion tests in the hbar limiter. 

- Bumping HBAR_RATE_LIMIT_TINYBAR prevents reaching the total budget when deploying large contracts by individual users. 

- Increased HBAR_RATE_LIMIT_BASIC to prevent batch1 from becoming flaky and encountering HBAR rate limit errors.